src/components/SchemaRow/TopLevelSchemaRow.tsx

```diff
@@ -4,11 +4,13 @@ import { Box, Flex, HStack, Icon, Menu, Pressable } from '@stoplight/mosaic';
 import { useUpdateAtom } from 'jotai/utils';
 import * as React from 'react';
 
+import { faExclamationTriangle } from '../../../node_modules/@fortawesome/free-solid-svg-icons/index';
```

Comment:
```suggestion
import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons/faExclamationTriangle';
```

Reply:
I updated the TopLevelSchemaRow import to match the SchemaRow import, but both had build errors if I didn't include the .js on the import so I went ahead and left the .js on there.

Reply:
Looks good!